### PR TITLE
Update tenant creation credentials JSON

### PR DESCRIPTION
**Pull request by jinapurapu**
**What does this do:** Changes tenant creation credentials JSON output to the format :
     {	
	      "url" : "http://localhost:5000",
	      "accessKey" : "YJ0RI0F4R5HWY38MD873",
	      "secretKey" : "OHz5CT7xdMHiXnKZP0BmZ5P4G5UvWvVaxR8gljLG",
              "api": "s3v4",
	      "path": "auto",
}

Addresses https://github.com/miniohq/engineering/issues/270

### CODE REVIEW

**Review by adfost — COMMENTED** [review]
Fails permissions tests.

File: operatorapi/consts.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ const (
 	ConsoleOperatorSAToken = "CONSOLE_OPERATOR_SA_TOKEN"
 	MinIOSubnetLicense     = "MINIO_SUBNET_LICENSE"
 	ConsoleMarketplace     = "CONSOLE_OPERATOR_MARKETPLACE"
+	ConsoleMinIOServer     = "CONSOLE_MINIO_SERVER"
```
Comment (harshavardhana) [inline operatorapi/consts.go:25]: This may be not present in console operator @jinapurapu - you need to get this information from the "tenant" spec.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: operatorapi/consts.go (hunk 1 above)
Comment (harshavardhana) [inline operatorapi/consts.go:25]: You need to remove this though @jinapurapu - this is not used in operator

**Review by harshavardhana — CHANGES_REQUESTED** [review]
Are we not planning to change this for Service Accounts UI? @jinapurapu

**Review by dvaldivia — APPROVED** [review]
Tested. LGMT.

**Comment by dvaldivia** [comment]
@harshavardhana this works on both regular `console` and `operator ui`

**Comment by harshavardhana** [comment]
> @harshavardhana this works on both regular `console` and `operator ui`

👍🏽

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**